docassemble/assemblylinewizard/data/questions/pdf_field_tester.yml

```diff
@@ -112,12 +104,12 @@ terms:
       with the name of a person/group of people, which we turn into "attributes" of that person.
 ---
 code: |
-  signatures = [{field: placeholder_signature} for field in signature_fields]
+  signatures = [{field: placeholder_signature} for field in signature_fields_test]
   checkboxes = [{field: 'Yes'} for field in checkbox_fields]
-  other_fields = [{field[0]: map_raw_to_final_display(field[0])} for field in fields if not field[0] in signatures and not field[0] in checkbox_fields]
+  other_fields = [{field[0]: map_raw_to_final_display(field[0])} for field in fields_raw if not field[0] in signatures and not field[0] in checkbox_fields]
   pdf_fields = other_fields + checkboxes + signatures
 ---
-mandatory: True
+field: pdf_check_display
 question: |
   Here is the filled-in PDF
 subquestion: |
```

Comment:
This question and subquestion need to be much more directed: what should the author be looking for when they go through this preview document? What should cause them to relabel and start over? Currently the screen is descriptive at telling you what the document has in it, but not what you should be watching out for, even if it's obvious to seasoned labelers.

Reply:
Good observation. See #391.